.circleci/config.yml

```diff
@@ -973,7 +976,7 @@ jobs:
             echo "Publishing draft release for wp-desktop $VERSION..."
             NAME="WP-Desktop ${VERSION#?}"
 
-            chmod +x desktop/bin/make-changelog.sh
+            chmod +x ./desktop/bin/make-changelog.sh
```

Comment:
🤔  This seems kinda strange, no? `chmod +x path/to/file` worked fine in the legacy repo, not sure why the `.` prefix is necessary here?